### PR TITLE
Include the RTPS/ROS2 bridge through Fast RTPS

### PR DESCRIPTION
**Pull request by VicenteEprosima**

### CODE REVIEW

**Comment by LorenzMeier** [comment]
The commits will need cleanup and we need to patch it up so it passes CI.

**Comment by hamishwillee** [comment]
Does this include configuration changes to start the "Client" as a permanent daemon process when PX4 starts up? If not, should we create a defect for that?

When done, can you point me to the section where this has been done so I can document (I am guessing that there will be a default configuration for the connection that users can replace via init file or environment variables to allow for different connections to companion computer/ROS development computer etc.).

File: README.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ The PX4 Dev Team syncs up on a [weekly dev call](https://dev.px4.io/en/contribut
   * Obstacle Avoidance - [Vilhjalmur Vilhjalmsson](https://github.com/vilhjalmur89)
   * [Snapdragon](https://github.com/PX4/Firmware/labels/snapdragon)
     * [Christoph Tobler](https://github.com/ChristophTobler)
-    * [Mark Charlebois](https://github.com/mcharleb) 
+    * [Mark Charlebois](https://github.com/mcharleb)
```
Comment (VicenteEprosima) [inline README.md]: Done 24812074339cb010f8dff29d2a16a22699185b72

**Comment by mcharleb** [comment]
@LorenzMeier @dagar The CI test platforms will need to have Fast-RTPS and Fast-CDR installed for the CI tests to pass.

**Comment by LorenzMeier** [comment]
@dagar Could you bring those libs in?

**Comment by mcharleb** [comment]
@VicenteEprosima Please merge https://github.com/eProsima/Firmware/pull/17

**Comment by mcharleb** [comment]
@LorenzMeier src/lib/micro-CDR needs to be excluded from check_format

**Comment by mcharleb** [comment]
@VicenteEprosima Please merge https://github.com/eProsima/Firmware/pull/18 to fix CI issues.

**Comment by mcharleb** [comment]
@dagar px4io/px4-dev-ros:v1.0 needs to have Fast-RTPS and Fast-CDR installed. They will have to be built from source because the binary download has a registration page.

**Comment by hamishwillee** [comment]
> They will have to be built from source because the binary download has a registration page.

@mcharleb @dagar  Not necessarily. There is a registration page to get the download link, but once you get there it is a hardcoded link (at least for windows). So we can script this if we like, but of course risk that the link will be broken in future. 
[and of course you'll get an email from eProsima the first time you register]

**Comment by LorenzMeier** [comment]
New PR: https://github.com/PX4/Firmware/pull/7685

**Pull request closed without merging**